backend/servers/express-webrouter/package.json

```diff
@@ -8,10 +8,13 @@
     "concurrently": "^3.1.0",
     "ejs": "^1.0.0",
     "express": "^4.14.0",
-    "nodemon": "^1.11.0"
+    "nodemon": "^1.11.0",
+    "svg-sprite": "^1.3.6"
   },
   "scripts": {
-		"compile": "babel --presets latest -d app/lib/ app/src/",
+    "compile-babel": "babel --presets latest -d app/lib/ app/src/",
+		"compile-sprite": "svg-sprite -d --di --defs-dest=app/templates --ds _svgs.html icons/*.svg",
```

Comment:
I would rather omit `sprite` than `icons`, i.e. if you want it short I would call this `compile-icons`.

Also, you're indentation here seems to be all over the place. :D